sympy/physics/mechanics/rigidbody.py

```diff
@@ -51,21 +53,24 @@ class RigidBody:
 
     """
 
-    def __init__(self, name, masscenter, frame, mass, inertia):
-        if not isinstance(name, str):
-            raise TypeError('Supply a valid name.')
-        self._name = name
-        self.masscenter = masscenter
-        self.mass = mass
+    def __init__(self, name, masscenter=None, frame=None, mass=None,
+                 inertia=None):
+        # This import will be removed with the introduction of InertiaTuple
+        from sympy.physics.mechanics.functions import inertia as f_inertia
```

Comment:
Would like to move the `inertia` function to a separate `inertia.py` file, when introducing the `InertiaTuple`. This will also remove some circular import problems.

Reply:
Agree.